apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -43,6 +44,7 @@ class UnitOverviewTopRow extends React.Component {
     courseOfferingId: PropTypes.number,
     courseVersionId: PropTypes.number,
     isProfessionalLearningCourse: PropTypes.bool,
+    publishedState: PropTypes.string,
```

Comment:
Because we know that this is not just a string but one of a known set, we can restrict this PropType even further to state "this has to be one of the PublishedStates". An example of that is here: https://github.com/code-dot-org/code-dot-org/blob/982bd54d36255d1a89c1824c2a8a28e12a934ee2/apps/src/lib/levelbuilder/unit-editor/UnitEditor.jsx#L46

Reply:
Thanks for this!  I appreciate that attention to detail!